experimental/datacommons-js/README.md

```diff
@@ -0,0 +1,366 @@
+    },
+  ],
+});
+```
+
+Renders:
+![Line chart](./assets/line-chart.png "Line Chart")
+
+### Map chart
+
+Draw a choropleth map
+
+Initialize empty div
+
+```html
+<div id="map-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawMap(document.getElementById("map-example"), {
+  id: "map-chart-1",
+  svgChartHeight: 200,
+  className: "my-chart",
+  apiRoot: datacommons.root,
+  isDataTile: false,
+  title:
+    "Population Below Poverty Level Status in Past Year in States of United States (2020)",
+  place: {
+    dcid: "country/USA",
+    name: "United States of America",
+    types: ["Country"],
+  },
+  enclosedPlaceType: "State",
+  statVarSpec: {
+    name: "Population Below Poverty Level Status in Past Year",
+    statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+  },
+});
+```
+
+Renders:
+
+![Map chart](./assets/map-chart.png "Map Chart")
+
+### Ranking chart
+
+Draw ranking chart (bar chart + ordered table)
+
+Initialize empty div
+
+```html
+<div id="ranking-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawRanking(document.getElementById("ranking-example"), {
+  placeName: "USA",
+  placeType: "State",
+  withinPlace: "country/USA",
+  statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+  isPerCapita: false,
+  scaling: 1,
+  unit: "",
+  date: "2020",
+});
+```
+
+Renders:
+
+![Ranking chart](./assets/ranking-chart.png "Ranking Chart")
+
+## API
+
+### `datacommons.drawBar`
+
+Draws bar chart tile with specified [statistical variable](https://datacommons.org/tools/statvar).
+
+- `element` (HTMLElement)
+
+  DOM element to attach the chart
+
+- `props` (BarTilePropType)
+
+  Chart configuration object
+
+  - `id` (string)
+
+    DOM id attribute to attach to the chart
+
+  - `title` (string)
+
+    Chart title
+
+  - `place` (NamedTypedPlace)
+
+    Scope the statistical variable query by this place configuration
+
+    - `dcid` (string): Place DCID
+    - `name` (string): Place name
+    - `types` (string[]): [Place types](https://datacommons.org/browser/Place)
+
+  - `enclosedPlaceType` (string)
+
+    Enclosing place type
+
+  - `comparisonPlaces` (string[])
+
+    A list of related places dcids to show comparison with the main place.
+
+  - `statVarSpec` (StatVarSpec)
+
+    Datacommons [statistical variable](https://datacommons.org/tools/statvar) configuration
+
+    - `statVar` (string): Statistical variable dcid
+    - `denom` (string): Denominator (e.g., `Count_Person`)
+    - `unit` (string): [Unit](https://datacommons.org/browser/UnitOfMeasure) (e.g., '$')
+    - `scaling` (number):
+    - `log` (boolean): Set to true to color values using logarithmic scale
+    - `name?` (string):
+
+  - `svgChartHeight` (number)
+
+    Height (px) for the underlying SVG chart.
+
+  - `className` (string)
+
+    Extra classes to add to the container.
+
+  - `isDataTile?` (boolean)
+
+    Whether or not to render the data version of this tile
+
+  - `apiRoot` (string)
```

Comment:
Maybe skip this apiRoot too and always use the default one for external library?

Reply:
agreed it should be optional, but i think we should keep it available in case someone is running a custom dc

Reply:
I updated the library to make this optional, but kept the param in the `example.html` examples for now until the next version of `datacommons.js` gets deployed